src/renderer/Presentational/ContentMultipleClients/ContentMultipleClients.tsx

```diff
@@ -175,37 +213,32 @@ const ContentMultipleClients = (props: {
       <HeaderMetrics {...(nodeOverview as NodeOverviewProps)} />
       <HorizontalLine type="content" />
       {renderPrompt()}
-      <div className={sectionTitle}>Ethereum Clients</div>
+      <div className={sectionTitle}>Clients</div>
```

Comment:
do we want this to be conditional so that with Ethereum clients, we show "Ethereum Clients" and any other types, Clients?

Reply:
To me, the Ethereum or even "<Node>" before this seems redundant as Ethereum is at the top of the screen @ddannehh 
<img width="1046" alt="Captura de pantalla 2023-09-18 a la(s) 3 59 38 p m" src="https://github.com/NiceNode/nice-node/assets/3721291/092c7534-9c29-4b5a-81b2-8607cab3353f">